phoenix-core/src/main/java/org/apache/phoenix/util/ViewUtil.java

```diff
@@ -629,4 +631,37 @@ public static void addTagsToPutsForViewAlteredProperties(List<Mutation> tableMet
 
         }
     }
+
+    public static ParseNode getViewWhereWithViewTTL(StatementContext context, PTable table)
+            throws SQLException {
+        ParseNode viewWhere = null;
+
+        Long nowTS = context.getConnection().getSCN() == null ?
+                EnvironmentEdgeManager.currentTimeMillis() : context.getConnection().getSCN();
+        String viewStatement = null;
+        String viewTTLFilterClause = String.format(
+                "((%d - TO_NUMBER(phoenix_row_timestamp())) < %d)", nowTS, table.getViewTTL());
```

Comment:
same here. If phoenix row ts is getting server-side ts, it's better to compare with server-side ts instead of client-side

Reply:
Will change here to use the CURRENT_DATE.

Reply:
+1, though maybe use `CURRENT_TIME` or `NOW` instead since that indicates datetime better than `CURRENT_DATE` (though all 3 may be doing the same thing).